packages/components/Organisms/HeroBanner/HeroBanner.vue

```diff
@@ -276,6 +277,7 @@ export default {
     &__title {
       @include rpl_typography_ruleset($rpl-hero-banner-title-typography-ruleset);
       @include rpl_text_color($rpl-hero-banner-title-color);
+      @include rpl_word_wrap($rpl-hero-banner-title-word-wrap);
```

Comment:
It just one line of simple css `word-wrap: break-word`, may not need to be a mixin. Any good reason for it?